src/shopping-cart.js

```diff
@@ -0,0 +1,28 @@
+import order from './data/order.js';
+import techs from './data/tech.js';
+import { findProduct, calcOrderTotal } from './register.js';
+import compileLineItem from './compile-line-item.js';
+import { toUSD } from './format.js';
+import shoppingCart from './data/order.js';
+
+const tbody = document.querySelector('tbody');
+
+listItems();
+// calc order total
```

Comment:
please delete comments like this before submitting - it's awesome to see that you're using them to guide your coding process, but you need to have them removed before submitting to get full points for clean code. I didn't take points off this time since it's not a consistent problem throughout your code, but please be mindful of it in the future!